### PR TITLE
Fix debian packaging for mesonbuild branch

### PR DESCRIPTION
**Pull request by danirabbit**
It looks like we need some changes to the debian packaging to keep things working

### CODE REVIEW

File: debian/install [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 plank/dock1 /etc/skel/.config/plank/
 profile.d/*.sh /etc/profile.d/
-sessioninstaller.desktop /usr/share/applications/
-settings.ini /etc/gtk-3.0/
+sessioninstaller/sessioninstaller.desktop /usr/share/applications/
+gtk/settings.ini /etc/gtk-3.0/
 sudoers.d/* /etc/sudoers.d/
 overrides/*.gschema.override /usr/share/glib-2.0/schemas
-.inputrc /etc/skel/
+inputrc/.inputrc /etc/skel/
```
Comment (worldofpeace) [inline debian/install]: shouldn't this file be completely unneeded https://www.debian.org/doc/manuals/maint-guide/dother.en.html#install

> If there are files that need to be installed into your package but your standard make install won't do it, put the filenames and destinations into this install file. They are installed by dh_install(1).

File: debian/rules [hunk 2]
```diff
@@ -3,5 +3,3 @@
 %:
 	dh $@
```
Comment (worldofpeace) [inline debian/rules]: I don't know a thing about debian packaging but I think we should copy https://github.com/elementary/switchboard-plug-pantheon-shell/blob/deb-packaging/debian/rules#L13

```suggestion
	dh $@ --buildsystem=meson
```

File: debian/install (hunk 1 above)
Comment (danirabbit) [reply debian/install]: It's necessary because we're generating multiple packages so we need to declare which package should contain which files

Comment (worldofpeace) [reply debian/install]: Ahh, I guess that makes sense then.

**Pull request merged**